src/test/java/com/codurance/retropolis/controllers/CardControllerTest.java

```diff
@@ -47,12 +51,10 @@
 
   @Test
   public void post_cards_should_return_back_card_instance_with_id_in_response() throws Exception {
-    Long cardId = 1L;
-    Long columnId = 1L;
-    String cardText = "hello";
-    String userName = "John Doe";
-    NewCardRequestObject requestObject = new NewCardRequestObject(cardText, columnId, userName);
-    given(cardService.addCard(any(NewCardRequestObject.class))).willReturn(new Card(cardId, cardText, columnId, userName));
+    String TEXT = "hello";
+    NewCardRequestObject requestObject = new NewCardRequestObject(CardControllerTest.TEXT, COLUMN_ID, USERNAME);
+    given(cardService.addCard(any(NewCardRequestObject.class)))
+        .willReturn(new Card(CARD_ID, CardControllerTest.TEXT, COLUMN_ID, USERNAME));
```

Comment:
this could just be `TEXT` rather than `CardControllerTest.TEXT` if you remove the declaration above